### PR TITLE
Updates to samples

### PR DESCRIPTION
**Pull request by PanosPeriorellis**
- added all regions to WPF sample
- added an object to parse the JSON on batch sample
- added properties to batch sample to batch URI parameters

## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[X] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
 # Linux executables
 quickstart/cpp-linux/helloworld
 samples/cpp/windows/console/samples/sample
+*.dll
+*.exe
+samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/objd/amd64/speechtotext-wpf.exe.config
```
Comment (wolfma61) [inline .gitignore]: >otext-wpf/objd/ [](start = 54, length = 15)

on top of the gitignrore file is a section to ignore build results, adding the objd there might be sufficient

File: .gitignore [hunk 2]
```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
...
+samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/objd/amd64/whatstheweatherlike.wav
+samples/batch/csharp/App.config
```
Comment (wolfma61) [inline .gitignore]: that file is part of the project? why do you want to exclude it?

File: .gitignore [hunk 3]
```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
 # Linux executables
 quickstart/cpp-linux/helloworld
 samples/cpp/windows/console/samples/sample
+*.dll
+*.exe
```
Comment (wolfma61) [inline .gitignore]: these are probably in the objd directory? so they should be excluded anyhow

Comment (PanosPeriorellis) [reply .gitignore]: you mean they would be excluded even without me explicitly saying so?

File: .gitignore (hunk 2 above)
Comment (PanosPeriorellis) [reply .gitignore]: no, accident. well spotted

File: .gitignore (hunk 3 above)
Comment (PanosPeriorellis) [reply .gitignore]: fixed gitignore as per wolfgang's instructions

**Comment by PanosPeriorellis** [comment]
#sign-off

**Pull request merged**